CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## 0.1: It Starts with a Single Die (2015-07-25)
 
 - :game_die: Support for rolling individual dice.
-- :game_die: Initial support for expressions (e.g. `2d8 + 8`). Greatly expanded support planned for the next release.
+- :game_die: Initial support for expressions (e.g. `2d8 + 8`). Greatly expand support planned for the next release.
```

Comment:
I fixed the other issues, but I guess this one I still think should be reduced to expand. Unless I am supposed to read this as 
> Initial support for expressions (which greatly expanded support planned for the next release)

 as opposed to two different but tightly related changes. Alternatively, it might be 

> Greatly expands support planned...

if we are talking about the effect of the initial support. ¯\\_(ツ)_/¯